<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuForm</class>
 <widget class="QWidget" name="MenuForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout">
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_start">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vertical_spacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_version">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
